src/katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -611,6 +611,13 @@
             "additionalProperties": false,
             "properties": {
                 "develop": {"type": "boolean", "default": false},
+{% if version >= "3.2" %}
+		"develop_opts":{
```

Comment:
```suggestion
		"develop_opts": {
```

Reply:
removed 'develop_opts' from schema, to simplify design as suggested